pkg/dbtoken/server_test.go

```diff
@@ -55,7 +55,7 @@ func TestServer(t *testing.T) {
 			wantStatusCode: http.StatusOK,
 		},
 		{
-			name: "GET unauthorized /token returns 403",
+			name: "GET unauthorized /token returns 405",
```

Comment:
this is reverting the change and the behavior is back to exactly what it was with gorilla/mux